classes/class-wc-rest-connect-service-data-refresh-controller.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+if ( class_exists( 'WC_REST_Connect_Service_Data_Refresh_Controller' ) ) {
+	return;
+}
```

Comment:
```suggestion
```
I know these checks are everywhere in other code but they are really unnecessary and new code should not include them. This file is included with require_once() so the class will never already exist. ABSPATH check is mostly used to prevent anyone from loading the file directly outside of WP context but if that happened nothing would happen because this file only declares a class.

Reply:
I agree. Although I'd like to clarify that the `class_exists( '...' )` check is made also to allow a rewrite of the class implementation from third party 👍 
I only added the checks for consistency with the existing code.

Reply:
>  Although I'd like to clarify that the class_exists( '...' ) check is made also to allow a rewrite of the class implementation from third party

That sounds like something we definitely don't want to encourage or support. If there was a case were we did we'd want to make use of interfaces and inheritance instead of overwriting entire classes. 

> Should we remove the if(!defined(...)) from all the other controllers?

Eventually we probably don't want to do it with this PR but I'd recommended it as part of a whole project update PHP code to new features like class autoloading.